platform/posix/utest/reconnect_utest.c

```diff
@@ -108,21 +120,49 @@ void test_Transport_ReconnectBackoffAndSleep_Succeeds( void )
 }
 
 /**
- * @brief Test that the value of the next max jitter is capped at some max threshold.
+ * @brief Test that the value of the next max jitter has a lower bound that will
+ * then be capped at some max threshold.
  */
-void test_Transport_ReconnectBackoffAndSleep_Jitter_Has_Max( void )
+void test_Transport_ReconnectBackoffAndSleep_Lower_Bound_Jitter_To_Cap( void )
 {
     bool retriesArePending;
 
     /* Initialize to 0 attempts, so the max value of next jitter will increase. */
-    reconnectParams.attemptsDone = 0;
+    reconnectParams.attemptsDone = 0U;
     reconnectParams.nextJitterMax = MAX_RECONNECT_BACKOFF_SECONDS / 2U;
```

Comment:
To truly test the lower bound of capping, this should use the lowest value whose double is greater than the capped value (i.e. `( MAX_RECONNECT_BACKOFF_SECONDS / 2) + 1`  )
```suggestion
    reconnectParams.nextJitterMax = ( MAX_RECONNECT_BACKOFF_SECONDS / 2U ) + 1U;
```

Reply:
In either case, what you end up testing is the same...